src/main/groovy/aspectj/AspectJPlugin.groovy

```diff
@@ -115,16 +119,28 @@ class AspectJPlugin implements Plugin<Project> {
 
 class Ajc extends DefaultTask {
 
+    @Internal
     SourceSet sourceSet
```

Comment:
Only works with `@Internal`, not `@Input` or `@InputFiles`. 

This may in fact be the reason why we have `com.signavio.gradle.StaleFileReportingBuildListener` in the SPM code base.